runelite-client/src/main/java/net/runelite/client/plugins/gpu/GpuPlugin.java

```diff
@@ -315,6 +321,13 @@ protected void shutDown()
 			client.setGpu(false);
 			client.setDrawCallbacks(null);
 
+			if (jawtWindow.getLock().getHoldCount() != NativeSurface.LOCK_SURFACE_NOT_READY)
+			{
+				jawtWindow.lockSurface();
+			}
+
+			glContext.destroy();
```

Comment:
This should probably be at the very end? Considering at startup it is before program/vao/texture alloc etc